### ISSUE TITLE
aquasec_enforcer_groups fails to create enforcer group of type 'kube-enforcer'

### ISSUE THREAD

**Issue by violiveira** [aquasecurity/terraform-provider-aquasec#124]
Hi all, I have tried to create an enforcer group of type kube-enforcer with the code below. However the provider fails with the following error:

`Error: {"message":"Currently  container runtime is not supported, supported container runtime is only docker for kube enforcer","code":400}`

As can be seen in the code, I am not even trying to set the 'container runtime' parameter. Maybe this parameter is hardcoded somewhere?

```
resource "aquasec_enforcer_groups" "kube-enforcer-group" {
  group_id = "kube-enforcer-group"
  type = "kube_enforcer"
  orchestrator {
    type = "kubernetes"
    master = false
    namespace = "aqua"
  }
  logical_name = "kube-enforcer-group"
  gateways = [ "aqua-gateway-csp" ]

  micro_enforcer_image_name = "registry.aquasec.com/microenforcer:6.5.22093"
  kube_bench_image_name = "aquasec/kube-bench:0.6.3"
  auto_copy_secrets = false
  micro_enforcer_secrets_name = "aqua-registry"

  enforce = false
  admission_control = true
  block_admission_control = false
  micro_enforcer_injection = false
  auto_discovery_enabled = true
  auto_scan_discovered_images_running_containers = true
  auto_discover_configure_registries = true
}
```

### Terraform Version
```
terraform -v
Terraform v1.1.5
on darwin_arm64
+ provider registry.terraform.io/aquasecurity/aquasec v0.8.10
```

### Affected Resource(s)
- aquasec_enforcer_groups

**Comment by yossig-aquasec** [comment]
Thanks for rasing this issue, we will check this and update you

**Comment by semyonmor** [comment]
@violiveira 
We fixed this issue and the fix will be available in release `v0.8.11`
